editor/audio_stream_preview.cpp

```diff
@@ -211,6 +213,16 @@ void AudioStreamPreviewGenerator::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("preview_updated", PropertyInfo(Variant::INT, "obj_id")));
 }
 
+void AudioStreamPreviewGenerator::_update_audio_streams() {
+	for (Map<ObjectID, Preview>::Element *E = previews.front(); E; E = E->next()) {
```

Comment:
I believe this can be [simplified](https://github.com/godotengine/godot/pull/50284).

Reply:
Done! Thank you.